ui/default.nginx.conf

```diff
@@ -31,6 +31,13 @@ gzip_types
     text/xml
     text/x-component
     text/x-cross-domain-policy;
+
+upstream model {
+    # Update to the cost model endpoint
+    # Example: host.docker.internal:9003;
+    server 0.0.0.0:9003;
```

Comment:
Most likely, deployed into the same pod as the cost model - therefore it should resolve to 0.0.0.0:9003.

Once we have the helm chart, we could update this to point at the service in the same namespace.